docs/GuidGuidRecommender.md

```diff
@@ -0,0 +1,144 @@
+
+# Overview
+
+The GUID-GUID coinstallation recommender system generates recommendations
+for add-ons which are similar to a given add-on,
+where similarity is based on the number of profiles
+which have both add-ons installed.
```

Comment:
We're into taste territory. To my eyes this is a little grandiose.

The GuidGuidCoinstalationRecommender Class generates recommendations, based on the co-installation rates of add-ons.

Reply:
I'd prefer to keep this as just a general introduction to what this is about. I feel like it's clear, if a bit long.